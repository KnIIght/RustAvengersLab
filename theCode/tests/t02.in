0
Tony
super intelligence
Mark L
