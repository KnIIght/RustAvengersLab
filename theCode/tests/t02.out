My name is Tony, my power is super intelligence and my weapon is Mark L.
I am Iron Man. *snap* 
In all due time. A hero becomes more than a hero.
